src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1021,9 +1041,9 @@ private static String storagedirFor(String type)
 
     private static String storagedir(String errMsgType)
     {
-        String storagedir = System.getProperty(Config.PROPERTY_PREFIX + "storagedir", null);
+        String storagedir = STORAGE_DIR.getString();
         if (storagedir == null)
-            throw new ConfigurationException(errMsgType + " is missing and -Dcassandra.storagedir is not set", false);
+            throw new ConfigurationException(errMsgType + " is missing and -D" + STORAGE_DIR.getKey() + " is not set", false);
```

Comment:
see? Here it is with `-D`

Reply:
Fixed.

Reply:
matter of taste, I dont have anything against _not_ specifying `-D` but we should be consistent in whatever we choose. Not only here. All similar cases where `-D` is added should be revisited.

Reply:
I mean, if the message says something like `please add -Dxxx=yyy`, then yes, there should be `-D`. However, if the message says `Please set property xxx to yyy`, then there should be no `-D`

Reply:
That make sense but raises the question: why do we have two different ways or saying the same thing?

Reply:
because this is OSS project and different people write the code? :)

Reply:
Fixed here and for other places matching the same pattern.